charts/osm-arc/templates/metrics-agent-deployment.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: osm-metrics-agent
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: osm-metrics-agent
+    {{- include "osm.arcLabels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: osm-metrics-agent
+  template:
+    metadata:
+      labels:
+        app: osm-metrics-agent
+        {{- include "osm.arcLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ .Release.Name }}
+      containers:
+        - name: mdm
+          image: linuxgeneva-microsoft.azurecr.io/genevamdm:master_20220401.1
```

Comment:
I would add images as fields in "values.yaml" and then reference them here. like how osm-label does for the Azure Policy / alpine image.